intdataplane/int_dataplane.go

```diff
@@ -326,6 +334,15 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 		dp.allIptablesTables = append(dp.allIptablesTables, t)
 	}
 
+	// Register that we will report a liveness indicator.
+	if config.HealthState != nil {
+		config.HealthState.RegisterReporter(
+			HEALTH_NAME,
+			&health.HealthReport{Live: true, Ready: true},
+			HEALTH_INTERVAL*2,
```

Comment:
Might be best to increase the grace period on the dataplane?  We've seen conntrack block for up to 12s (although we mean to fix that).